bugstats.php.dd

```diff
@@ -43,6 +43,6 @@ $(P <center>$(LINK2 https://issues.dlang.org/reports.cgi?product=D&datasets=NEW&
 Macros:
   TITLE=The D Bug Tracker
   PHP=<?php $0 ?>
-  BOOKTABLE = <center><table cellspacing=0 cellpadding=5 valign=top class=book><caption>$1</caption>$2</table></center>
+  BOOKTABLE = <center><table cellspacing=0 cellpadding=5 valign=top class="book"><caption>$1</caption>$2</table></center>
```

Comment:
Ack, my bad, didn't notice at first this was documenation, not the actual macro definition.  But in any case, shouldn't it be updated to match the actual current definition?